.github/workflows/trivy-security-scan.yml

```diff
@@ -1,15 +1,40 @@
 ##
-# This action runs trivy container and repository vulnerability
-# scanner for docker images and cargo packages.
+# This action runs Trivy container and repository vulnerability
+# scanner for Docker images and filesystem.
 ##
 
 name: trivy-security-scan
 
 on:
   repository_dispatch:
-    types: [ trivy-scan-dispatch ]
+    types: [trivy-scan-dispatch]
+  schedule:
+    - cron: '0 0 * * *'  # Runs daily at midnight UTC
```

Comment:
GitHub says that schedules at the start of the hour are more likely to be skipped under load:
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule

```suggestion
    - cron: '26 0 * * *'  # Runs daily after midnight UTC
```

Reply:
We don't need the cron schedule